src/constants/queryParams.ts

```diff
@@ -23,6 +23,6 @@ export const QUERY_PARAMS = {
   // Reports
   author_type: "at",
   // Pass through
-  "concept_filters.id": "cfi",
-  "concept_filters.name": "cfn",
+  concept_id: "cfi",
+  concept_name: "cfn",
```

Comment:
This might need renaming back too

Reply:
Ah, got it - we don't need to.

Every layer of transformation we add, we expect our future selves to remember we made, and what is often harder to work out in a year, why we made it.